### ISSUE TITLE
JSON Schema needs updates

### ISSUE THREAD

**Issue by PatStLouis** [w3c/vc-data-model#1577]
There are 2 updates I would like to add to the [json schema](https://github.com/w3c/vc-data-model/blob/main/schema/verifiable-credential/verifiable-credential-schema.json) file:
- remove the [required `credentialStatus.id` property](https://github.com/w3c/vc-data-model/blob/a78384a16fe9d6ac33e4fc2adfa2a44aa309fe74/schema/verifiable-credential/verifiable-credential-schema.json#L54), as this field is not required in the `BitstringStatusList` specification.
- remove the [`proofChain` property](https://github.com/w3c/vc-data-model/blob/a78384a16fe9d6ac33e4fc2adfa2a44aa309fe74/schema/verifiable-credential/verifiable-credential-schema.json#L320), as proof chains are represented in the proof property.

**Comment by msporny** [comment]
Yes, please make those changes. We really do need to bring the JSON Schema file up to date with the latest specification. If you find anything else wrong in the JSON Schema file, please let us know.

**Comment by msporny** [comment]
PR #1581 was raised to address this issue. That PR has been merged, closing.
